rmw_fastrtps_shared_cpp/src/TypeSupport_impl.cpp

```diff
@@ -78,11 +78,12 @@ bool TypeSupport::serialize(
         eprosima::fastcdr::FastBuffer fastbuffer(  // Object that manages the raw buffer
           reinterpret_cast<char *>(payload->data), payload->max_size);
         eprosima::fastcdr::Cdr ser(  // Object that serializes the data
-          fastbuffer, eprosima::fastcdr::Cdr::DEFAULT_ENDIAN, eprosima::fastcdr::Cdr::DDS_CDR);
+          fastbuffer, eprosima::fastcdr::Cdr::DEFAULT_ENDIAN, eprosima::fastcdr::CdrVersion::XCDRv1);
```

Comment:
CI is complaining that this line is too long (needs to be <= 100 characters): https://ci.ros2.org/job/ci_linux-aarch64/14925/testReport/junit/(root)/projectroot/cpplint/

Reply:
9f0296e